test/JDBC/expected/single_db/db_accessadmin-vu-verify.out

```diff
@@ -0,0 +1,337 @@
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the user 'dbo'.)~~
+
+DROP ROLE db_owner
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the role 'db_owner'.)~~
+
+DROP ROLE db_accessadmin
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the role 'db_accessadmin'.)~~
+
+DROP USER guest
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: User 'guest' cannot be dropped, it can only be disabled. The user is already disabled in the current database.)~~
+
+
+-- db_accessadmin can not add/drop members to any other role
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the role 'db_accessadmin', because it does not exist or you do not have permission.)~~
+
+ALTER ROLE db_owner ADD MEMBER babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Adding members to db_owner is not currently supported in Babelfish)~~
+
+ALTER ROLE babel_5136_r1 ADD MEMBER babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Current login babel_5136_db_accessadmin_l1 does not have permission to alter role babel_5136_babel_5136_r1)~~
+
+
+-- should not be able to alter users login mapping or rename members of db_owner
+ALTER USER babel_5136_u1_new_name WITH LOGIN = babel_5136_l3
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Current user does not have privileges to change login)~~
+
+ALTER USER dbo WITH LOGIN = babel_5136_l3
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the user dbo)~~
+
+ALTER USER dbo WITH NAME = dbo_should_not_be_renamed
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the user dbo)~~
+
+ALTER USER dbo WITH DEFAULT_SCHEMA = dbo
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the user dbo)~~
+
+GRANT CONNECT TO babel_5136_u1_new_name
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+GRANT CONNECT TO dbo
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+REVOKE CONNECT FROM babel_5136_u1_new_name
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+REVOKE CONNECT FROM dbo
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+
+-- NO OTHER DDLs SHOULD BE ALLOWED
+CREATE TABLE babel_5136_t1 (id INT)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for schema dbo)~~
+
+CREATE PROCEDURE babel_5136_p1 AS SELECT 1
+GO
+~~ERROR (Code: 2714)~~
+
+~~ERROR (Message: Function 'babel_5136_p1' already exists with the same name)~~
+
+CREATE FUNCTION f1() RETURNS INT AS BEGIN RETURN 1 END
+GO
+~~ERROR (Code: 2714)~~
+
+~~ERROR (Message: Function 'f1' already exists with the same name)~~
+
+CREATE DATABASE new_database
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Only one user database allowed under single-db mode. User database "babel_5136" already exists)~~
```

Comment:
This is not intended output. Please update.

Reply:
Updated